Headers/README.md

```diff
@@ -1,40 +0,0 @@
-MacKernelSDK
-============
-
-This project lets your kernel extensions (kexts) target a broad range of XNU kernels when using the latest Xcode version right from the interface. In addition it also includes several headers only available in other projects or older Xcode releases.
-
-## Quick start
-
-To start using **MacKernelSDK** do the following steps:
-
-1. Clone MacKernelSDK to your your project directory:
-
-    ```sh
-    git clone https://github.com/acidanthera/MacKernelSDK
-    ```
-
-2. Add custom `KERNEL_EXTENSION_HEADER_SEARCH_PATHS` and `KERNEL_FRAMEWORK_HEADERS` values in project configuration in Xcode. To do this choose **Add User-defined Setting**.
-
-    ```
-    KERNEL_EXTENSION_HEADER_SEARCH_PATHS = $(PROJECT_DIR)/MacKernelSDK
-    KERNEL_FRAMEWORK_HEADERS = $(PROJECT_DIR)/MacKernelSDK
-    ```
-
-    ![xcode-config](Docs/xcode-config.png)
-
-3. Optionally add `/MacKernelSDK` to `.gitignore`.
-
-## Targeting `i386`
-
-To compile for 32-bit you will need to add a set of flags in your project:
-
-1. Other C Flags (`OTHER_CFLAGS`):
-
-    - `-nostdlib`
-    - `-target i386-apple-macos10.6`
-    - `-fallow-unsupported`
-
-2. Other Linker Flags (`OTHER_LDFLAGS`):
-
-    - `-static`
-    - `-target i386-apple-macos10.6`
```

Comment:
You dropped a pic below, which is used in README. Please restore.

Reply:
I dropped the entire README in Headers, which I thought was redundant. Should I add it back?

Reply:
The picture, named `Headers/xcode-config.png`, is used by the other `README.md`. I cannot comment explicitly on it.